examples/with-google-analytics-amp/lib/gtag.js

```diff
@@ -1,9 +1,9 @@
 export const GA_TRACKING_ID = '<YOUR_GA_TRACKING_ID>'
 
 // https://developers.google.com/analytics/devguides/collection/gtagjs/pages
-export const pageview = (url) => {
+export const pageview = () => {
   window.gtag('config', GA_TRACKING_ID, {
-    page_path: url,
+    page_path: window.location.pathname + window.location.search,
```

Comment:
Is this missing the window.location.hash 🤔

Reply:
Good question, I took inspiration from [`react-ga`](https://github.com/react-ga/react-ga/blob/e82caf57f0c797bc801db6057cfc3ba43a1a3a2d/README.md#with-npm) for this.